shared/js/l10n.js

```diff
@@ -1226,8 +1226,8 @@
     var createEntry = this.isPseudo() ?
       createPseudoEntry.bind(this) : Resolver.createEntry;
 
-    for (var i = 0, node; node = ast[i]; i++) {
-      this.entries[node.$i] = createEntry(node, this.entries);
+    for (var i = 0; i < ast.length; i++) {
+      this.entries[ast[i].$i] = createEntry(ast[i], this.entries);
```

Comment:
yeah, I know, but I prefer to fix it now than having to come back to it later. Added build/l10n part